nifi-nar-bundles/nifi-ocr-bundle/nifi-ocr-processors/src/test/resources/tessdata/tessconfigs/nobatch

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this supposed to be an empty file or one that was created (which is probably better located in target/)?